sympy/solvers/solvers.py

```diff
@@ -1253,6 +1253,7 @@ def _solve(f, *symbols, **flags):
     If no method is implemented to solve the equation, a NotImplementedError
     will be raised. In the case that conversion of an expression to a Poly
     gives None a ValueError will be raised."""
+    f = simplify(f)
```

Comment:
I know you found that the routine works if you add this, but it would be better to find what is failing. simplification is only attempted later when other methods don't work (iirc).